R/tm_g_heat_bygrade.R

```diff
@@ -396,21 +397,20 @@ srv_g_heatmap_bygrade <- function(id,
     output_q <- shiny::debounce(
       millis = 200,
       r = reactive({
-        ADSL <- data[[sl_dataname]]() # nolint
-        ADEX <- data[[ex_dataname]]() # nolint
-        ADAE <- data[[ae_dataname]]() # nolint
+        ADSL <- data()[[sl_dataname]] # nolint
+        ADEX <- data()[[ex_dataname]] # nolint
+        ADAE <- data()[[ae_dataname]] # nolint
 
         teal::validate_has_data(ADSL, min_nrow = 1, msg = sprintf("%s contains no data", sl_dataname))
         teal::validate_inputs(iv(), iv_cm())
         if (isTRUE(input$plot_cm)) {
           shiny::validate(shiny::need(all(input$conmed_level %in% ADCM[[input$conmed_var]]), "Updating Conmed Levels"))
         }
 
-        qenv <- teal.code::new_qenv(tdata2env(data), code = teal::get_code_tdata(data))
```

Comment:
`qenv `is also referenced in after the  if statement. due to this, unchecking the` plot conmed `option in the example app's sidebar will cause the application to fail. Could you also  replace `qenv` with `data()` on line 434 as well.

Reply:
Thanks for catching and fixing this. I totally missed this one.